app/assets/javascripts/components/locales/en.jsx

```diff
@@ -94,7 +94,6 @@ const en = {
   "navigation_bar.public_timeline": "Federated timeline",
   "notification.favourite": "{name} favourited your status",
   "notification.follow": "{name} followed you",
-  "notification.mention": "{name} mentioned you",
```

Comment:
new IntlMessageFormat(intlMessages[`notification.${notification.type}`], intlLocale)

Ah, I see. I will revert this line. Thanks!